docs/kb/api-version-4.md

```diff
@@ -1,48 +1,98 @@
-# API Version 4
+# What is Version 4?
 
 In order to accommodate a new feature related to prospects,
 we have created a new version of our APIs - version 4.
 We are now allowing multiple prospects to share an email address on some Pardot accounts.
 If your account has this feature active now, then you MUST use version 4.
 Everyone else can continue using version 3.
```

Comment:
"Otherwise you can continue to use version 3"  Should continue to talk directly to the user.